org/wildfly/mod_cluster/capability.adoc

```diff
@@ -0,0 +1,19 @@
+NAME: org.wildfly.mod_cluster
```

Comment:
I've added all the requirements it has. TBH it is a mess how mod_cluster & capabilities are used.